assignment/src/com/assignmentweek2/person/Person.java

```diff
@@ -3,11 +3,8 @@
 This is the abstract class
  */
 public abstract class Person {
-        abstract void Studentdisplay();
-        abstract void Teacherdisplay();
-        abstract void Doctordisplay();
-        abstract void StudentAddress();
-        abstract void TeacherAddress();
-        abstract void DoctorAddress();
+        abstract void display();
+        abstract void Address();
```

Comment:
method name should start with small letter.

Reply:
changed